app/presenters/download_dropdown_display.rb

```diff
@@ -100,14 +102,24 @@ def button
                 "aria-expanded" => "false")
   end
 
+  def link
+    content_tag("a",
+                "<i class='fa fa-download' aria-hidden='true'></i> Download".html_safe,
+                class: "dropdown-toggle download-link",
+                id: menu_button_id,
+                "data-toggle" => "dropdown",
+                "aria-haspopup" => "true",
+                "aria-expanded" => "false")
+  end
```

Comment:
Is `link` exactly the same code as `button` but for the first parameter of `content_tag`? Maybe one or two more? But a lot in common?

I wonder if it would be better to have `link_or_button` (or some other name), that just picks it's arguments to `content_tag` that vary appropriately, while leaving the ones that don't vary alone. 

That way we don't have to copy things like `data-toggle` and `aria-haspopup` in two places, and make sure to change them in two places if they change.

Reply:
Yup - thanks for catching this.